string isim
float64 konumX
float64 konumY
float64 konumZ
float64 linearHizX
float64 linearHizY
float64 linearHizZ
float64 angularHiz
